apps/src/StudioApp.js

```diff
@@ -453,6 +453,18 @@ StudioApp.prototype.init = function(config) {
       // contract editor
       return true;
     }.bind(this));
+    Blockly.contractEditor.registerDialog(function (bodyText,
+        confirmText, cancelText, onRightButton, onLeftButton, leftButtonClass) {
```

Comment:
Would it make sense to pass in an options param instead of all these individual params? Makes it easier to see what callers are doing.

Could potentially even have these options be the same as those sent to showSimpleDialog, and then do
```
Blockly.contractEditor.registerDialog(this.feedback_.showSimpleDialog.bind(this.feedback_, this.Dialog));